main.js

```diff
@@ -46,14 +46,15 @@ function parseInput() {
 }
 
 async function downloadInstaller() {
-  const inst_path = tc.find('msys2-installer', inst_version, 'x64');
+  const version = `0.0.${inst_version.replace(/-/g, '')}`
```

Comment:
What's the problem with the follwing?

~~~js
const version = `${inst_version.replace(/-/g, '.')}`
~~~

Reply:
Using the last part only makes the version incompatible to all other versions according to semver.

Reply:
Thanks! I will add a comment about it!

Reply:
good idea. (though I don't know if anything does any semver related version comparison here, but that's one advantage I could think of)

Reply:
what I was thinking is that if msys2 ever wanted to use 'real' versions, version numbers up to 2021 (or even newer) would be considered 'older' than version 1.0.

I do think there is some semver comparison going on in the tool-cache too.